src/Symfony/Bundle/FrameworkBundle/Translation/PhpExtractor.php

```diff
@@ -105,13 +129,34 @@ private function seekToNextRelevantToken(\Iterator $tokenIterator)
         }
     }
 
+    private function seekToMessageParamsEnd(\Iterator $tokenIterator)
+    {
+        $openBraces = 0;
+
+        for (; $tokenIterator->valid(); $tokenIterator->next()) {
+            $t = $tokenIterator->current();
+
+            if ('[' === $t[0] || '(' === $t[0]) {
```

Comment:
@aitboudad In your example, `$t` will be an array consisting of three elements where the value at index 0 is `T_CONSTANT_ENCAPSED_STRING` if I am not mistaken.